posthog/session_recordings/test/test_session_recordings.py

```diff
@@ -225,8 +226,39 @@ def test_session_recordings_dont_leak_teams(self) -> None:
         response = self.client.get(f"/api/projects/{self.team.id}/session_recordings")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         response_data = response.json()
-        self.assertEqual(len(response_data["results"]), 1)
-        self.assertEqual(response_data["results"][0]["id"], "2")
+
+        assert response_data["results"] == [
```

Comment:
this test seems to fail intermittently locally... but since it was just comparing array length there was no useful output when it did